catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -346,7 +332,22 @@ protected Object executeWithSubject() throws Exception {
   }
 
   private File getInputFile() {
-    final File inputFile = new File(filePath);
+
+    final File inputFile;
+
+    if (StringUtils.isBlank(filePath)) {
+      inputFile = getDefaultIngestLocation();
+      if (!inputFile.exists()) {
+        printErrorMessage("No file or directory provided and no default directory exists.");
```

Comment:
✏️ Should we say `no default directory exists` here?

Reply:
Matt and I discussed. Leaving as-is